src/Containers/Rockets/Rockets.jsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import Card from './Card/Card';
 
-const Rockets = () => <div>Rockets</div>;
+const Rockets = () => {
+  const RocketData = useSelector((state) => state.rockets);
+  return (
+    <>
+      {RocketData.map((data) => (
+        <Card
+          img={data.flickr_images}
+          key={data.id}
+          title={data.rocket_name}
+          desc={data.description}
+        />
+
+      ))}
+    </>
```

Comment:
- [ ] **SUGGESTION:** Instead of using the react fragment component `<></>`, It would be a great idea to wrap this component with a `<section>` tag, this way we keep the semantics of our document meaningful and improve the overall accessibility of our page.